index.html

```diff
@@ -3,12 +3,12 @@
     <head></head>
     <body>
         <script type="text/javascript">
-            
+
             console.log("location: ", window.location);
 
-            window.location.href = "http://localhost:8000/packages/node_modules/sample-browser-auth-implicit/" + window.location.hash;
-        
+            window.location.href = "http://localhost:8000/packages/node_modules/samples/browser-auth-implicit/" + window.location.hash;
```

Comment:
Should we get the domain/port from window location and use it instead of localhost?